PeerioTechnologies/peerio-mobile
# chore(medcryptor): revert loading screen [branch ch16831]

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/16831/mc-mobile-disable-new-animation-for-mc
#### Testing instructions  
Verify that Medcryptor's loading scren has been reverted back to the old one.
Verify that the new Peerio loading screen is still intact.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
